app/models/post.rb

```diff
@@ -1,3 +1,16 @@
 class Post < ApplicationRecord
   belongs_to :user
+  has_many :likes
+  has_many :comments
+  after_save :update_posts_counter
+
+  private
+
+  def update_posts_counter
+    user.increment!(:posts_counter)
+  end
+
+  def recent_comment
+    comments.order(created_at: :desc).limit(5)
+  end
```

Comment:
- [ ] NIce job on making this method that returns most_comment of the post, however this method (`recent_comment`) should not be private because it will be accessible outside of the Post model.  kindly remove it in private methodes.

Reply:
Thanks..it have been resolved